checkups/kubevirt-vm-latency/automation/e2e.sh

```diff
@@ -166,7 +168,8 @@ if [ -n "${OPT_DEPLOY_CHECKUP}" ]; then
     echo "Deploy kubevirt-vm-latency..."
     echo
 
-    ${KIND} load docker-image "${CHECKUP_IMAGE}" --name "${CLUSTER_NAME}"
+    ${CRI} save -o "/tmp/vmlatency-image.tar" "${CHECKUP_IMAGE}"
+    ${KIND} load image-archive --name "${CLUSTER_NAME}" "/tmp/vmlatency-image.tar"
```

Comment:
I saw a [nice trick](https://github.com/kubernetes-sigs/kind/issues/2729#issuecomment-1110152680):
```suggestion
    "${CRI}" save "${CHECKUP_IMAGE}" | "${KIND}" load image-archive -
```

I will remove the need to save the image file to disk (thus no need to remove it).

Reply:
It doesnt work for me: 
```
$ podman save quay.io/kiagnose/kubevirt-vm-latency:devel | ./kind load image-archive --name kind 
ERROR: name of image archive is required
```

Reply:
Too bad.
Thank you for trying.
I've tried as well and also failed.

Reply:
Could you please put the temporary filename in a variable and delete the file after the load to kind?

Reply:
Done